src/me/EtienneDx/RealEstate/Config.java

```diff
@@ -18,6 +23,7 @@
 
     public final String configFilePath = RealEstate.pluginDirPath + "config.yml";
     public final String logFilePath = RealEstate.pluginDirPath + "GriefProtection_RealEstate.log";
+    final static String messagesFilePath = RealEstate.pluginDirPath + File.separator + "messages.yml";

```

Comment:
The FileSeparator is already included in the plugin dir path

```suggestion
    final static String messagesFilePath = RealEstate.pluginDirPath + "messages.yml";

```